### PR TITLE
Add links to Navbar | Improved formatting 

### LINKED ISSUE #110: Updation of Nav-Bar and other fixes
- Nav bars are needed to be updated on the website with the pages of Speakers as well as tickets. 
- Convince my Boss is to be added to the Share bar Ribbon or drop-down menu (Discuss)

### PR DESCRIPTION
**Pull request by vipulgupta2048**
The review might be difficult due to the amount of the changes done. Hence I am summarising them here. 
- Deleted many extra lines and improved formatting making .html files look more uniform, 
- Fix #110 #130 
- Added Js source files to convince-my-boss.html
- Didn't fix sponsorship.html since not in use. 
-  In Volunteer.html changes are due to fixing of spaces and formatting by Atom Beautify.

Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipulgupta2048@gmail.com>

### CODE REVIEW

File: coc.html [hunk 1]
```diff
@@ -2,225 +2,216 @@
...
+                    <span class="icon-bar"></span>
+                </button>
+            </div>
+
+
+            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+                <ul class="nav nav-justified">
+                    <li><a href="index.html#home">Home</a></li>
+                    <li><a href="index.html#journey">Journey</a></li>
+                    <li><a href="#speakers">Speakers</a></li>
```
Comment (ananyo2012) [inline coc.html]: `index.html#speakers`

File: coc.html [hunk 2]
```diff
@@ -2,225 +2,216 @@
...
+                    <!-- <li><a href="#conference">Schedule</a></li> -->
+                    <li><a href="index.html#sponsors">Sponsors</a></li>
+                    <li><a href="#tickets">Tickets</a></li>
```
Comment (ananyo2012) [inline coc.html]: `index.html#tickets`

File: convince-my-boss.html [hunk 3]
```diff
@@ -58,9 +58,10 @@
                 <ul class="nav nav-justified">
                     <li><a href="index.html#home">Home</a></li>
                     <li><a href="index.html#journey">Journey</a></li>
-                    <!-- <li><a href="index.html">Speakers</a></li> -->
+                    <li><a href="index.html">Speakers</a></li>
```
Comment (ananyo2012) [inline convince-my-boss.html]: `index.html#speakers`

File: convince-my-boss.html [hunk 4]
```diff
@@ -58,9 +58,10 @@
...
                     <!-- <li><a href="index.html">Schedule</a></li> -->
                     <li><a href="index.html#sponsors">Sponsors</a></li>
+                    <li><a href="#tickets">Tickets</a></li>
```
Comment (ananyo2012) [inline convince-my-boss.html]: `index.html#tickets`

**Comment by vipulgupta2048** [comment]
@ananyo2012 Changes made as requested. 
I was testing the links, then it hit me. We have to update the blog as well. Making a PR.

**Comment by ananyo2012** [comment]
@vipulgupta2048 So I merged some PRs. The changes contained fixing the navbar issue for convince-my-boss, adding favicon to the page and also the sponsor contained moved to javascript. Please rebase your changes over the latest master and solve the conflicts. I know this will be a lot of work. Let me know if you need any help.

**Comment by vipulgupta2048** [comment]
Test the website here https://vipulgupta2048.github.io/inpycon2018/

**Comment by ananyo2012** [comment]
The download buttons are not looking that great. Can you make a separate PR for that. This is ready to be merged.

**Comment by vipulgupta2048** [comment]
@ananyo2012 Sure, removing this commit. I took the buttons from the ticket table so it looked uniform.

**Pull request merged**